tests/sdks/python/application/test_secret_validation.py

```
@@ -0,0 +1,86 @@
+        # Act & Assert — no exception expected
+        validate_secrets(secrets)
+
+    def Should_Raise_When_AnyValueIsEmpty(self) -> None:
+        # Arrange
+        secrets = {
+            "DB_URL": "postgres://localhost",
+            "API_KEY": "",
+        }
+
+        # Act
+        act = lambda: validate_secrets(secrets)
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `act` as a `def`

(E731)